### DISCUSSION TITLE
Change offset ms by ms instead of 5 by 5 or fractioned while pressing a button

### DISCUSSION THREAD

**Discussion by Hexality** [ppy/osu#19052] [Ideas]
Just like the title says.

Basically, having to change the offset is steps of 5ms isn't practical and changing with the pen can be pretty difficult, so why not adding a button that while changing the offset it disables the fractional numbers.

https://user-images.githubusercontent.com/17398632/178083006-9943f690-a7a9-49d5-b3c8-2a099d673124.mp4
> ALT: Almost impossible to precisely change the offset

**Comment by frenzibyte** [comment]
This seems to be the only slider which shifts by large magnitudes, unlike all other sliders which shift by precise fractions.

**Reply by Hexality** [reply]
Both global and local offset sliders do this

**Comment by fyyrraaa** [comment]
maybe having the slider allow for precise adjustments by pressing alt/option while dragging it would be good as well; i know at least in Lightroom when doing colour grading with a mouse, dragging the cursor in the colour wheel while pressing alt/opt makes it easier for more precise selections

**Reply by Hexality** [reply]
Like, the game already do change from fraction to 5 by 5 when pressing shift and moving the mouse, but I don't think 5 by 5 should be the way.

**Reply by fyyrraaa** [reply]
i think i was way too tired when i read that at first, now i understood what you meant correctly! my bad!

**Comment by peppy** [comment]
I think this should use a similar control to the metronome / timing control in the editor to allow more precise adjustments. I've brought this up with @arflyte so let's see what he comes up with.
